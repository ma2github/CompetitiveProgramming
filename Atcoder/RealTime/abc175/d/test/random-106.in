1853 254340478
370 1473 1415 628 1848 1089 951 135 1216 1362 1129 467 52 1201 72 608 770 1592 1481 716 1247 792 1424 20 1413 424 267 431 1347 1246 564 1775 1213 692 401 602 1828 787 429 204 974 390 815 256 484 802 534 1650 1794 995 1839 1718 890 506 510 1416 1580 493 944 278 1541 1824 601 570 1666 336 948 1128 61 1531 776 543 1664 1599 1427 1783 1224 1163 547 268 1805 178 83 1506 195 1092 930 198 1536 806 1107 835 1074 567 1596 668 670 993 1736 1826 1227 1389 1000 260 250 928 1207 840 521 1681 373 1714 168 1176 509 1113 1737 1618 1547 647 1619 1406 882 977 1050 291 1310 980 1202 1261 296 51 337 989 1386 1395 258 494 87 1495 875 184 372 1651 925 965 795 206 1674 1630 1356 225 587 116 1033 92 387 579 861 1324 1391 593 1184 590 667 307 77 645 1276 233 427 728 586 551 964 1766 91 201 1492 1682 1265 845 1221 1245 272 1849 1267 67 952 919 341 1136 807 1041 1801 741 755 1468 1643 79 1212 295 469 1670 1111 1713 1432 604 1064 896 563 1689 634 1663 461 782 1625 881 1068 10 1108 1155 636 1097 1256 936 1821 414 724 859 808 731 1355 1399 160 617 1411 753 1036 1628 737 276 749 629 836 382 175 437 1287 927 671 154 739 488 1364 1806 752 428 990 561 366 979 996 1703 1066 1259 1270 1657 661 113 923 1098 577 1293 251 1587 1781 435 319 1283 1035 678 367 292 1215 244 1046 1403 140 1048 600 1527 887 662 239 183 935 1537 1709 789 71 16 326 1616 856 1466 1127 1106 1313 1023 426 378 1711 1648 1483 612 391 420 654 1563 1842 1712 1123 1617 19 762 926 1529 1 1748 350 1690 1059 1357 1467 1119 148 1306 982 1353 1459 26 413 685 1390 219 1581 1331 448 1392 369 1725 1656 144 623 3 915 1085 279 616 374 1316 1497 511 669 801 883 1233 1503 851 37 99 542 1520 1405 1338 631 904 813 780 706 1026 156 1093 352 262 1295 161 496 290 1027 850 1515 1708 1436 1181 828 1669 707 294 1597 1191 192 186 1220 1430 1115 719 810 1153 1572 299 438 1381 81 1825 1410 1382 1082 1438 1028 1194 681 1684 841 269 181 1193 1333 1465 742 1037 163 1798 1765 1214 334 1589 1598 672 1178 1571 1799 1441 1125 536 1822 1697 910 1710 1156 641 409 1613 109 119 1796 1278 1417 137 356 1185 215 85 477 1706 6 884 918 389 1594 1302 1606 1636 1414 1602 489 1102 1271 1196 106 1776 734 868 595 804 1502 1463 1084 1109 1501 55 688 1004 945 649 907 1006 800 643 1604 170 1052 124 7 754 182 1361 1695 976 571 1397 684 1641 1236 456 1379 929 591 803 1314 847 750 187 1819 1354 1705 1218 1062 589 867 1735 1363 252 1813 149 1055 957 78 453 227 1388 1686 1375 153 1662 1767 562 371 263 1431 811 1197 412 853 1600 878 660 1150 1069 627 1070 1551 531 1365 823 1189 482 1100 1540 869 1542 637 1294 1500 1340 960 283 443 1786 65 1508 1146 1426 1015 228 1073 903 1607 261 1474 1815 245 769 572 588 763 938 45 101 1464 906 1750 1060 285 1260 626 1829 528 1565 1275 1784 514 190 954 565 1667 1014 120 911 632 640 1292 541 1554 895 1522 500 970 1509 1321 1516 398 1544 1768 725 403 491 1831 1121 130 343 1680 1462 693 720 1758 864 1264 1677 554 576 1533 1448 128 1001 757 909 199 440 1228 1234 498 656 1030 914 211 8 481 1377 1484 597 1521 580 1144 415 932 777 664 54 1428 809 449 550 1174 702 1746 487 108 21 1665 1320 714 1297 760 522 1075 695 1319 1830 1844 492 1058 436 1507 538 689 464 1137 379 88 1723 1514 146 1005 1384 214 1770 1707 687 726 98 633 1610 1440 147 796 308 35 1754 64 1254 470 535 1170 1654 155 1110 582 1024 1002 790 955 1240 679 248 169 677 97 818 173 410 1159 1237 90 1687 730 969 60 1635 82 1332 288 1273 1727 355 1101 281 829 984 421 1559 1836 1255 1683 1691 508 29 1717 1841 1470 962 949 286 265 235 422 179 872 396 423 722 1300 1299 1549 665 257 93 23 418 1077 933 1204 1504 1755 457 102 138 1751 1167 1372 746 697 1791 1548 62 519 833 1672 832 333 1226 1230 348 1771 1434 1203 111 1359 344 1845 167 1419 100 1788 1134 1007 1132 1019 819 1348 22 1383 736 799 998 765 1367 606 451 432 568 425 873 1139 698 594 217 210 1461 177 238 1289 392 658 553 1182 1032 363 1243 213 1840 1743 1472 444 708 886 559 1760 1420 1649 196 1763 1517 1157 1471 1490 507 1574 441 520 978 59 1479 216 1774 1122 1173 1535 581 1360 1539 727 103 271 439 110 180 394 1779 1371 690 1570 1699 1510 48 1640 797 303 309 1172 1637 53 1513 1235 891 1345 501 1460 473 598 613 317 1493 221 1211 934 1443 902 259 327 305 1615 638 894 1149 1437 1543 652 243 433 50 1777 1716 166 793 1352 1346 651 1407 362 274 468 849 1733 452 821 1135 1833 1693 1511 1188 1154 231 1812 1790 691 569 917 253 1335 1020 1279 610 1291 1526 1118 912 1229 1422 529 1120 150 408 1401 546 220 768 1021 1730 1584 1742 1787 406 1179 47 377 992 384 723 1258 205 1117 1626 1063 230 1206 953 223 1369 1446 1327 1694 525 1720 400 188 1852 284 460 1478 383 503 302 1696 1621 991 497 471 232 96 342 1042 1668 1116 1088 901 321 1016 298 1631 1850 1568 1647 1632 185 331 1688 842 718 63 1595 417 157 848 293 1095 1272 1162 826 345 1318 1087 1645 1408 122 611 717 874 174 249 699 1263 860 224 376 15 504 1591 846 620 176 226 916 817 703 715 1453 931 1433 1336 339 322 1652 242 275 1582 676 1518 133 1795 218 1183 1456 478 1378 607 1409 1701 1130 961 1199 712 744 335 164 129 879 1053 1423 480 264 172 1288 1807 1124 1114 1044 502 766 619 545 1452 1739 325 609 1094 18 838 486 946 1575 1160 445 788 1759 871 696 1143 783 537 41 1593 237 549 959 557 381 1724 1722 705 1400 358 313 1609 1853 1166 865 13 791 194 1477 574 1342 889 131 673 686 203 820 430 12 1280 655 39 540 114 289 446 1744 1343 1091 1169 1671 1608 1368 527 1588 573 1315 1282 635 1567 1764 442 66 775 476 1586 123 1753 434 24 666 202 738 280 368 310 729 1145 1298 1262 544 4 1576 771 966 1175 1846 329 1398 1447 1673 1239 1309 1304 605 675 1553 566 1274 1561 399 1769 986 880 459 1330 1305 1496 1349 1081 899 1418 1809 855 1624 1112 1721 1785 466 1008 622 1351 159 1003 1661 397 17 1692 764 618 1039 1546 683 1450 1322 386 1161 483 1105 1043 126 38 1242 1334 136 596 1676 921 994 1222 31 1685 625 1489 784 411 659 1719 351 393 346 523 963 89 822 1210 134 1494 304 1491 958 208 1200 1080 888 1797 25 1290 1284 897 1065 1524 642 80 950 273 1017 1009 1761 834 1738 1457 1578 513 971 767 388 56 320 973 1425 1366 1103 1285 1217 158 1072 1376 1455 404 740 653 1429 306 940 1301 947 1177 1449 615 621 512 1133 1732 885 1803 983 1741 340 1219 330 32 454 324 1079 862 141 1679 1601 314 1010 1469 630 1045 857 1454 657 1528 644 721 1475 772 402 1678 1482 1104 1022 222 1086 858 207 1231 1499 505 1556 968 142 1451 779 558 1734 475 578 756 694 781 1555 458 359 844 837 143 1660 1350 1778 1286 824 75 1208 69 94 1165 1612 1303 1099 774 997 1704 1056 1238 639 277 455 57 1225 816 682 1729 1817 1012 603 450 11 1018 1344 118 332 1585 1047 1257 1808 713 852 1341 1747 34 1394 759 1834 73 1370 1049 987 360 287 49 560 241 1569 270 778 1557 1658 877 533 407 624 1577 981 1773 479 1393 353 104 1168 1253 1071 1485 318 1029 1789 941 1620 876 585 866 316 1646 1534 364 773 1476 1251 920 463 614 1545 1311 70 312 1140 1281 905 805 1252 1385 1223 924 1374 648 922 524 1827 1629 999 46 1605 139 1622 1762 956 827 1090 1025 747 761 1083 1820 1148 893 1523 27 1838 465 1772 1138 575 9 1728 42 1126 1442 1634 830 14 1550 1811 1373 1488 1487 165 1538 1603 1655 937 197 532 1698 518 162 1269 1038 5 315 117 1312 1152 74 1326 33 1187 1519 674 247 474 592 1232 499 898 1564 328 1337 700 967 1505 84 814 447 2 1804 1756 1186 1458 1823 297 1011 240 1659 646 1051 786 1268 1061 115 485 1308 900 1702 1847 86 1402 548 1096 1793 1530 1818 1444 68 266 1421 1380 599 1307 151 1726 943 1792 28 1835 354 1250 908 419 191 127 1040 112 1171 357 1067 1180 1445 1800 939 246 43 1266 58 988 913 556 1782 361 1627 44 1731 1244 710 338 892 1558 709 1642 972 1486 1439 1614 825 1241 583 1329 495 1412 711 95 584 125 1198 975 145 843 1802 1387 349 1435 1814 1205 300 311 380 1740 794 1583 1076 1158 1700 526 1249 1810 539 189 1031 758 152 385 1780 704 1480 1566 1745 323 1552 132 515 839 193 1034 1532 1757 212 405 105 751 36 1147 1190 745 748 209 76 200 985 663 1296 555 1131 870 229 236 1562 733 552 171 395 107 942 1832 255 516 1623 30 1675 1404 254 490 121 1639 1498 1525 1590 732 1339 347 1057 1633 1816 1317 1192 701 1638 1358 301 863 1837 854 650 1644 1715 1328 831 743 812 462 375 416 1579 735 1141 282 1164 517 1151 1323 1325 1013 530 1843 1078 1142 1209 1749 1653 798 1611 1054 40 1248 234 1573 472 1851 1560 365 680 1512 1396 785 1195 1752 1277
-159327197 -334936591 -683903172 -575230976 -797506228 493315216 -745469277 55056423 161646159 -312191312 227538313 103549286 133433225 272254578 -23860573 -524654590 806428037 -750763306 -202215057 305084516 -473049282 -874019016 60068396 -279614927 940353804 -492635251 341597782 842197751 38407773 -847614512 -774250235 -267610621 -114324728 -885913081 683172092 678386193 636110705 -965473795 -878662425 472045405 747055965 -750385623 -767860343 -512592276 545338518 560937223 -404868672 906897625 -979537259 -730968558 98523429 -102780643 -169231303 318819036 -918221950 330723446 58437538 -433632105 -559254685 -654997749 -62976834 -990744906 227612560 -307476145 -774177151 -887600760 878183380 -655926564 459305375 538683398 676069480 -440504409 -795422806 -425620482 28849679 -55631500 628307048 -331072208 633089226 -372517671 -9049325 -730139661 548993989 -624158579 -701323131 -951252736 -769334534 38775424 751955854 -379853643 162820703 964585392 161893445 674845269 215060557 476019388 587469943 -572387829 -482324621 -798649483 -894564213 579225685 -177060866 912212538 -859016884 -390989519 540568185 -635327941 -882285801 -465478484 -650557369 -698241615 -998703081 309382191 991097171 -629544514 252117604 -341703300 45914752 -771831191 -92546493 269589894 246899302 563548270 -953886534 -801528103 388233942 981447470 113814490 -795393039 -228480159 -378165037 61165465 -14318115 -990132137 604934235 618381950 -322925640 -73172397 -213759780 -257187107 552722249 -544473929 615805982 -69240954 248527909 -43321071 567594878 -851312581 -864633513 820055093 135038074 513117654 -290085659 174232918 596478839 84777606 -556636907 932731790 62018363 -414502703 -854507687 748997862 555080827 417665228 870206378 -924762840 865710727 -236610728 -807020082 997799509 -881051964 -136739208 23795688 324543527 830919339 687921944 -455328457 517029714 -310883196 796246020 -605095836 881164768 -113210872 -874974419 43146394 895579615 -761758536 -751243625 -118924146 -547825158 -633276504 -681235819 823752533 -59538382 -830470700 574721010 128407234 -885642613 -757553779 -896224217 -195081807 263219447 313485924 789426698 -607573882 -353985025 -321748717 -65896631 328613362 557507074 -824544585 80446603 629739396 790234050 688458053 -194974550 -703966288 557506238 692910539 -473117642 205328890 -475105997 379875761 644485227 -739037196 -267217594 848097785 241496665 -673972305 795118834 -181363621 -537558889 -137227111 593371478 -246746868 143424285 223436956 -149971434 -395446299 900987123 -781461884 457551815 -965342038 884262315 767161263 -644202006 213802838 -897264592 262941064 -365118484 819077272 -183590172 -762452822 -580750001 -636847283 -644767664 -681640166 180052821 592251348 -822113756 18148656 9434151 623983084 -777067654 100045441 362238449 700606304 589252903 706095151 -345822454 546123782 548929584 -947805484 -590550520 708327606 425723726 906941316 -412930258 910680720 422804079 932441362 485158138 974385939 -750911042 -620563078 746674669 -209739969 -26895509 -485476610 577987977 -619407384 227385475 -95528271 -132131962 -917014243 208466590 697836827 -423297480 409060466 112938253 -287589814 172136387 149342876 -413877531 -753681152 -914835593 586194803 524896636 372154612 616688740 -672301956 -349908861 631030790 271944862 215384239 -602466272 -322358468 585825888 251825825 785600904 179200360 23107245 -635245070 -218272570 -582285835 381916469 33095499 577430556 462798185 941883632 963828202 188523227 326984713 104656316 821603874 101626506 -756503882 -472676397 -626680557 602617285 -136250389 -188000645 -198025135 325561014 -968203977 654657045 -986163371 -365014544 -330839293 928945001 516335920 564315953 -82022324 248174061 -454368548 -369290612 978462021 442498772 -944787824 563725037 -791027132 359084800 -40590151 -144388591 105009805 352741055 577504751 -123555052 -629858762 -171040448 -528742290 -268094436 311746662 -537759348 318585552 -334745240 13443845 522887653 -675765386 -877322204 111834083 -280814314 251317581 -904681655 571184798 -210548326 -125181005 608228514 21015948 742322138 -376583863 666665406 501601414 -115400167 503496062 528229005 622585527 397767898 895305208 202920245 208975565 -628583615 150245231 -113966688 590412414 -376188059 -494115173 -578931571 -687985555 384143615 535100716 473785307 -411328472 -121241839 -243093363 -471590136 701103199 172954708 650288791 942847422 110227065 616368025 -452217417 -220889476 -414159515 448001449 -836483019 878145762 685289146 921133031 191339680 -732765227 -503702001 449525792 -829473028 163123105 -530976157 -43785760 -518569902 -297094144 480714554 920490454 -754987574 -72569869 -21286696 482248087 -458892426 -736412681 -707762847 88811618 -990512598 -6684480 76255035 -521763305 -940659493 -495121000 111517978 -819882657 897754745 -738423045 -554639960 -840400975 -437520229 -269411029 927518809 -891475755 -662884986 -465175034 -142039768 176572966 -570441405 -454886623 -4918700 135106714 -97050243 736463565 -614323186 169407459 929388183 447431219 -332404627 224204155 -210189554 -277760758 78545285 933949039 535039511 789587195 -49226709 593497394 -220698041 384441462 -209364654 -408549729 -231310170 46799541 -230791795 146832513 101825192 30248149 -330957525 -285279741 -709327293 -354988859 312349735 -171013649 557223403 953883043 -733270473 -598479339 616470013 -237972810 190844953 -791901567 243929438 -334136750 -757520709 -101559240 -668994945 507854688 -414818426 -70659838 528433314 17876980 -664944903 -96151802 150301758 -331044 84697400 490789229 -972274597 400193984 987703653 126368674 -730019775 -282865241 -722779785 75155550 970681841 -2709847 -110031769 344308621 20383480 -229476397 108452278 230184128 -698899408 117392374 610137225 -511457057 499205751 -754071960 790401613 -709449428 264459 627732975 517241344 -679298125 -144897152 708386072 875521275 -196511930 -776019774 436965219 -471293354 141962539 -490014925 -194548281 -689209566 -699896532 775864142 535647508 -300485537 -830025112 -870402731 -172336852 -234804602 -423460148 -275264890 359748346 -280475604 -422846379 322209124 841706471 480788232 -689403410 -92649857 223432448 927088545 -293130420 548217581 642365430 -231434030 579241348 897956399 569477343 -984844921 513349204 -881174257 872606260 -174179775 146232 -98571510 -62290414 -317933382 132942813 -546538680 873125080 -359105921 206659418 993222496 170748552 697667207 649448132 666235831 -510694565 -770492336 755824866 950126285 -190842039 -20512878 648327273 -804094523 785454034 278909797 -609036497 946733243 -999237726 -496287104 951177128 457848623 913286203 268293070 -431389753 -306475391 470462874 927081924 -194448312 536406161 -446115902 -467150547 -166210916 -706106789 737922249 -909472761 -534137320 -573806160 -360734206 -797810011 668581197 109427601 -172749206 -873605075 -62076812 -170990896 -34076327 15006221 522878976 -345617180 -595702860 308763143 511466008 106843523 244708019 373552514 802883271 -606107248 210710463 790572801 28392010 -862004738 -544962004 99784060 615006596 -257449253 69850117 627402170 703149160 -234608084 -625976515 841830643 -363092569 251213438 964503683 164676642 -563282509 811716112 998610145 -677444528 -735873320 -191009456 160352951 881240194 -591716379 -905993651 593503016 -294215977 807907205 329480197 371218899 -957751447 875827137 -958358162 20994978 -771930962 -203652810 -74965043 563505479 938942142 115691662 -199030104 -774205251 -589660892 -884410825 -371113426 922351253 520029739 976937109 529674227 -544089605 999757773 594607358 -437787858 -768822657 -209850658 807042908 831857801 -913050048 -639815037 249392302 531538335 561875835 -547900898 -944925471 -976635369 -419000411 -519072378 727314550 78710859 969812426 -798115832 -853873215 -842165340 -912720482 696247697 723445838 840148089 -630384615 -86669472 -70257682 235927888 -486298111 -197399348 -50739565 -768174711 -321885899 -311538490 -35277428 145858110 43062910 635110153 -544409835 -81855551 44283488 281662684 942367455 -59802418 -324651659 385177263 586117392 -542447682 -683134726 342968039 473473852 948286709 -784686876 405826149 220270118 15539486 -602514832 574014747 -682199334 -584934188 77624814 973430148 757189928 929628482 -645278139 78889387 23745701 302634876 -930167745 105060674 816514421 297832455 -566391302 -579402133 -837420426 -182470072 -964640631 407348626 -57896514 387976728 -696751086 492897947 661461033 -353485605 302437746 145602111 805600956 743762759 -659011435 -876690011 -473742516 616845698 716998447 145336828 158328780 -222684125 -393378590 585888787 -547847201 -407141724 -471428896 659932345 28913348 474516597 607042288 832069820 -883748818 416803187 -365751812 -117227265 -831791044 -468513269 525879208 630337090 -480025862 249271847 40705247 77327026 348083557 -160505311 148250739 -222990696 -459634798 -989111649 111242331 -228179188 846145143 -873412333 -235456156 -112555635 689515436 -557687484 -165186527 -528315670 -569127199 763541940 58305485 258952910 83029780 728390283 -193225432 -217207423 138073468 388242637 479669475 644673824 515347509 -862241494 540326740 -492795057 60970033 833354074 376980499 896447199 -299997241 975182096 122074677 649266103 -901472934 420775290 499870115 -974801107 322520333 -259122412 98041907 -755486196 326480552 -281748688 540877731 -829409182 -509412560 939677665 950814895 846944091 658136741 -753037391 -948862801 -140076318 -710294033 169825837 231170000 -758690134 409304144 -301907002 525519070 -239196821 812687755 -939553988 -193237788 -769044809 -875941446 82120404 757203736 -620339945 -820762019 576522731 493562928 160280372 530363537 98997161 -591582532 -417521807 696323245 759695026 379467959 945548326 807888797 57298032 638371736 858953671 547854587 -697206810 242374629 -142608391 -495114747 -761586624 964251032 -232721767 -403001674 -49470429 -995373459 846562558 874341629 83439823 -959507095 62161385 518177220 631391292 -738161631 -665506787 -487293056 -48024070 -297565502 -284354177 329004057 533060313 -255201813 -138797599 148094180 -128343598 -53154360 809817563 289249205 -601323868 575108532 -656069269 -176937234 559370277 595191186 753521690 410446745 827453075 673857338 478674255 184861348 -862800127 -189173785 133052846 -37594863 713957821 -376923230 338681051 880929886 -84240069 -752371745 108468629 -177181552 617161535 398417611 -625318002 306759339 -491297563 930073095 45272691 -250497023 380372846 -310392568 166415766 -438554419 -443565744 -404385208 570701435 -379660303 -5981978 -468282349 -418577765 -553965254 -572419197 329609893 767013742 -933522421 181558547 -174708636 232063266 652084912 468702097 104993256 60740870 213809932 -633441047 201900966 -486049422 -189684842 -242161778 904382838 -692769537 321765308 481069704 856955268 438840643 43214274 292791204 280686771 102849688 366271844 -852374919 130077277 -6461878 131570884 726049132 483424188 381123600 87031910 360696914 -756374895 -465249268 435418728 -494255806 156892684 877070297 32667439 733145909 -697693511 -686967370 -649121460 979508635 -816869963 358720867 -483784921 235994417 222670411 -228879032 73210180 881657449 572987174 941320676 209497661 670075799 542167603 518299009 169425619 -847881483 -71372342 -242590714 -997055004 -261779680 -217915714 -219483965 -695193973 345003736 134476444 -953723253 -167711028 616071916 152225114 -112867764 -249237409 -377669142 929583096 769180423 -337907669 336964709 -327703232 -510005183 -921590646 -490741892 243619451 -46360114 154606491 49640272 471372488 393088301 385083319 -999989074 -203661865 356556210 -74663461 666068047 -251813055 531282904 -173874495 503231404 -856070477 -150779342 938133227 155466697 -883324591 890734122 113822495 523470128 -568541792 -744458962 -325443386 47590312 -771449586 -504924945 -532201617 -100556211 73896963 -661855580 958318171 189056740 230377474 -197121128 37944952 -938244130 -679522639 -203737849 482630177 836650908 822676462 617678655 937495094 -425974362 -816879887 -519231021 -624520123 -11189727 731396236 374038642 -57923494 -574652047 562458411 -755965217 764879683 851784732 -300970261 -629900910 784209058 -109728225 762752164 870472128 234451903 -369091432 598644737 -114070213 -476896068 -761131622 303253439 123049785 7657975 -844981040 -374506689 759972243 -241710666 356058779 -676787285 120380974 -263868560 386994772 855726370 -842705087 -318909486 -26991319 454840414 -358698462 -559806552 31566514 605079552 395731594 397854162 362039030 499538044 -74615823 -43006232 -100338406 748948158 237810391 648547834 -956490707 -885272730 94389358 845198789 425200992 917514757 -359032250 38470982 687649494 924478097 -470855958 563048800 -221574802 966343963 602322034 -993019679 -855359889 680746233 531361112 -634231728 -98876371 54863028 -422394489 -414854703 809895476 -231546253 -184185605 -185334233 820508866 -12225884 684175824 -379377957 918752429 -356045401 -167981899 94903656 -290531046 47615274 393062145 690448077 -679033803 550801014 854921521 1507749 404785119 -382026750 -417939972 34785095 862684658 391163217 739550325 947999798 777799575 -957270880 902105212 338634844 -638534134 136637052 346609768 -851455202 216668042 -384787888 184126720 382081886 -313513089 -528846191 173089566 -41877919 338841993 432409556 -257063395 540999456 -357751240 -155946085 84728645 -982869895 -218496880 196219153 -2131697 -617794666 -607456592 279258333 -795404765 10696401 -288029059 -324225580 -518614513 294304229 978246719 -204773275 232296979 -771872332 482428616 340176853 366557077 305446817 277426260 -456263719 3634690 474476564 -972647102 -410290518 -384032339 259006171 -852737506 62090177 -707588277 147206575 -608644666 -986459443 -687346126 -671594487 -955572096 -754735665 720745046 13932588 959578848 -628502311 -534522120 922664983 941304383 -951640577 -519653047 -994714488 -265201819 401732309 967452137 -137315356 -692300063 148964911 -814609752 -54365893 -90893272 143722265 992210818 -220311560 -898856642 -354510374 983524879 740034996 652400974 969879043 756019006 163136695 -469454730 -954668062 -355011854 -770820062 669906278 984883788 -697750464 381924881 -830801411 -70088074 911748343 167106932 -8581646 142659788 -835632045 517793805 906574612 180511111 808135731 -635326070 281574085 -137128770 520203128 -406620264 521400328 175811449 315859462 20383420 83070923 -872284626 -494179427 472039122 -533389409 542719539 688788145 -44683031 -450960129 -298809053 254167895 -612488798 801800101 784436516 -820905250 -781410406 838610435 458423230 -737359335 617674600 406846902 578146662 -422162163 640966015 -569758806 -951231795 458336877 -317723598 -997265000 529385004 901016723 871718965 454196909 -521292776 -364033122 -583579983 -833018397 -681083083 -579162580 136903060 73474867 668234159 799616993 528122489 246102880 339687748 -712453340 -271546565 -548694240 -279416475 992810176 -655336888 -762593104 990711810 -359695169 -214309221 -958335906 633171512 -260774774 236091030 454817453 -403500055 -510274792 766846504 -593930492 647828884 213084565 -489639257 823810836 682527992 -202845334 360157045 -994216568 -383562591 463110224 -731874571 -297416152 335315705 -286172990 804735276 480679166 972892732 -191583463 -952169883 282273 942081487 -510784691 -754336286 47564562 -471672994 672751348 -608671834 203023489 606853721 244210800 -8846876 -239119601 -100960899 -505894147 61281417 -558594740 386020006 281138965 49389614 677909818 -513426149 737130762 824431203 137260446 -999749819 -997521049 -677050005 449417590 665387761 -141738894 -318395626 761272322 -138967156 -228250963 -525315324 599408643 509098292 884292765 -649739762 99706370 257694468 958418966 788806991 204780295 -367126512 476162793 364774589 175374017 292249588 661153240 -448960865 -170057897 344922751 -654847278 -756047623 639863461 665664548 -968270878 -687723327 -231074177 270226489 -243912522 -171626171 683542191 -556194411 -431565732 -56055097 -780423923 -285615982 -792766521 424354946 254629392 515880152 322823771 294993298 89634644 -822663595 292980084 -281181790 -37020399 -999821402 -801116363 -339458043 -581944229 -398655586 694342109 -818875664 -807197371 933356137 -475416084 -960579363 900089736 697845189 -947347392 226374758 876470957 -109135381 484350859 209869643 -509939126 -790229744 -61735129 319185760 -512812559 880448464 741640284 748482945 845523565 147031542 927816084 127909398 557747660 580446705 901939405 -181232731 266514685 -969874538 -312343689 975937836 645138682 -209129174 -960910961 -109763726 752275796 -14109925 -75605597 881200140 995027730 -692216012 891047914 118583702 -395378383 -29318693 -332641001 937092721 643606951 -467927026 196563382 485537966 502841914 107815682 -284331673 -980825922 -742180637 -820055918 910414506 527833156 373826514 -919626796 19929411 275807566 -321481207 -53456682 986666738 -181749563 162720243 -260762802 547179757 -640210683 -910204482 -644564058 -296368777 725302140 44641120 295726671 -765061107 26002295 -780793468 334641638 729382101 734067477 181750376 -44844304 976472412 -496328009 -778688244 -556236689 306266423 -565897845 506016803 -975464381 -603964144 885165454 74277716 850137429 323361276 -599369040 986219289 698416995 -612217739 -564986381 851813794 -80971994 310442364 536401790 -690316572 258391499 -319053333 -475803105 -831995018 714100487 -226334098 -468813965 -103320760 646269180 415299699 231208088 616161534 -306804897 55039418 418142301 -815926126 458368444 -104834849 -254605156 -348577819 -286096974 932683393 -320260385 -834347364 56115767 106698226 586267049 955458949 -756807294 615508378 392341398 -720272696 -567628606 -238135566 896724459 -884205038 -999161682 -21003960 -791531436 -338384236 -319742087 841861460 438353233 -258393930 -31387792 365743567 480675363 792160773 573405104 -707849943 -356469788 358472899 -225860245 34312667 33617839 -535470723 -336319313 -801710280 -370826426 -63184648 -53606632 228348958 -190271007 730105339 928433873 -304902943 500374998 128400976 -218290614 428012116 -584192027 403786571 317897083 -802197942 -696088930 -738342001 -172163343 -790585721 -326393408 90171180 -379617756 654731562 931789295 480127369 560378788 557998130 -801262473 628353560 585838040 546336061 480827295 -438727650 257240856 -791303036 365498839 -951738663 767385371 -208365229 563344262 441973392 -953438490 728353669 116483803 329693393 -337717924 585613896 108852528 627310109 -663583440 -915392367 283622046 -627552295 -963180583 -283872312 265399138 -614466492 -462550758 531051712 388311326 690970796 974924413 -168804420 84777233 2601158 527284201 -485132569 -556675322 388400679 527904322 958253333 807532069 -653779345 647312976 816483091 -311235944 -957073149 669472241 659831654 666892098 209771464 -333313630 -885470805 -759993466 837571149 806074509 -760709213 645386492 137173811 206308116 -920220704 -182411587 30465754 -317576872 -57723118 693260651 639389307 -540444426 -388903930 -401688929 138072923 228890374 -405328202 -557738302 623778945 833673755 -637278451 717615234 520211896 504662337 -615203512 -760507126 905960987 -916362514 -739160935 574362085 392636119 -765603526 -981953139 710439104 884784964 -737024613 474155082 -130336570 957470750 480717006 866530706 297764862 -982998656 -38471374 -106772885 208181559 908627998 56327067 125040786 767202113 -882313060 -392198509
