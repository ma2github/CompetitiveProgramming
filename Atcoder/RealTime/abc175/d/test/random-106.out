139631393581386742
